src/aind_behavior_curriculum/behavior.py

```diff
@@ -0,0 +1,45 @@
+"""
+Base Behavior Models
+"""
+
+from pydantic import BaseModel, ConfigDict, Field
+
+
+class AindBehaviorModel(BaseModel):
+    """
+    Defines Pydantic configurations applied to all behavior models.
+    BaseModel: Validate arguments on initialization.
+    Configurations:
+        - extra='forbid':
+            Do not allow a model to be initalized with undocumented parameters.
+        - validate_assignment=True:
+            Revalidate fields against schema on any change to model instance.
+        - strict=True:
+            Enforce strict typing.
+    """
+
+    model_config = ConfigDict(
+        extra="forbid", validate_assignment=True, strict=True
```

Comment:
can we add `validate_defaults` to the base class too?

Reply:
updated